src/app/test_executive/block_production_test.ml

```diff
@@ -17,7 +17,7 @@ module Make (Engine : Engine_intf) = struct
   let run network log_engine =
     let open Network in
     let open Deferred.Or_error.Let_syntax in
-    let block_producer = List.nth network.block_producers 0 in
+    let block_producer = Caml.List.nth network.block_producers 0 in
```

Comment:
Not necessary, of course, but wanted to make clear the existing call wasn't from `Core_kernel`, which returns an option.

Since we know this will succeed, changing it to `Core_kernel.List.nth_exn`.